doc/changelog.d/575.changed.md

```diff
@@ -0,0 +1 @@
+CI - 574 - Update codecov action to v4
```

Comment:
I suppose we should add this directory to an exclusion? It's irritating, because in general it's good to spellcheck these things, but we don't want to be adding individual exemptions for individual packages.

Reply:
I raised https://github.com/ansys/actions/issues/477 - It's going to happen to everyone, we should do it centrally

Reply:
What's the approach for getting this PR merged then, are we blocked until that ticket is resolved?

Reply:
A splendid question, we could update our exclusion regex, but I don't know if you can exclude based on paths, or only on filenames. I wonder what pyMapdl is doing

Reply:
https://github.com/ansys/pymapdl/blob/42ef51351b0b3db0ff8a7c3cc84406abd07c6fe9/doc/.vale.ini#L37C1-L37C16 one solution I suppose

Reply:
Don't we already have the Vale.Terms = No included?

Reply:
Yes, I subsequently realised this, so I'm puzzled. Theres no exclusion in their pre-commit config file that I can see

Reply:
Maybe we just add `codecov` as an allowed word